unicodetools/data/linkification/dev/LinkEmail.txt

```diff
@@ -1331,4 +1332,4 @@ FFDA..FFDC     # 1.1   [3] (ￚ..ￜ)  HALFWIDTH HANGUL LETTER EU..HALFWIDTH HAN
 3D000..3FC3F   # 18.0 [11328] (U+3D000..U+3FC3F)  SEAL CHARACTER-3D000..SEAL CHARACTER-3FC3F
 E0100..E01EF   # 4.0 [240] (U+E0100..U+E01EF)  VARIATION SELECTOR-17..VARIATION SELECTOR-256
 
-# Total code points: 162119
```

Comment:
I think this delta might be because of v18 characters.

Not sure what is going on, because main doesn't seem to have the environment variable that was in Robin's PR, that lets us switch between v17 and dev.

Reply:
You need to put -DPOST_SYNCHRONIZED_17 on the maven command line invocation of the generator to generate the 17 files rather than the dev=18 ones.